### PR TITLE
module-tools test run

### PR DESCRIPTION
**Pull request by brianjmurrell**
Update module-tools.

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

### CODE REVIEW

File: include/travis/mock_build_test.sh [hunk 1]
```diff
@@ -37,6 +37,8 @@ groupadd --gid "$(stat -c '%g' "$MAPPED_DIR")" mocker
 useradd --uid "$(stat -c '%u' "$MAPPED_DIR")" --gid "$(stat -c '%g' "$MAPPED_DIR")" mocker
 usermod -a -G mock mocker
 
+yum -y install https://rpm.nodesource.com/pub_8.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
+yum -y install nodejs
```
Comment (jgrund) [inline include/travis/mock_build_test.sh]: Are we going to add this for all repos?

Comment (brianjmurrell) [reply include/travis/mock_build_test.sh]: It's a WIP still.

**Comment by jgrund** [comment]
Is this ready for review / land now?

File: .npmignore [hunk 2]
```diff
@@ -1,14 +1,22 @@
-/.vscode
```
Comment (jgrund) [inline .npmignore]: Can we just switch to a whitelist instead?

File: .travis.yml [hunk 3]
```diff
@@ -36,14 +36,10 @@ jobs:
       if: branch =~ ^v\d+\.\d+\.\d+-.+$
       env: Type='Copr deploy'
       before_deploy:
-      - openssl aes-256-cbc -K $encrypted_253525cedcf6_key -iv $encrypted_253525cedcf6_iv -in include/copr-mfl.enc -out include/copr-mfl -d
-      - curl -O https://raw.githubusercontent.com/m3t/travis_wait/master/travis_wait
-      - chmod 755 travis_wait
+      - include/travis/copr-deploy.sh prepare
```
Comment (jgrund) [inline .travis.yml]: Nice.

**Pull request merged**